web/src/components/table/use-cases/traces.tsx

```diff
@@ -363,6 +371,7 @@ export default function TracesTable({
           />
         );
       },
+      enableSorting: true,
```

Comment:
@RichardKruemmel how would sorting work here? As far as i understand, the API does not support that yet.

Reply:
We can leave this out for now and solve this in another PR so we get this merged